mg.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Memory Game</title>
+    <link rel="stylesheet" href="mg.css" />
+  </head>
+  <body>
+    <div class="game-board">
+      <button class="card"></button>
```

Comment:
This is fine for the first week, but creating the button using DOM manipulation will make your life easier for the following weeks.